genesis-tools/genesis-csv-to-json/src/main.rs

```diff
@@ -18,9 +20,23 @@ fn main() {
                 .takes_value(true),
         )
         .arg(Arg::with_name("chain-id").long("chain-id").takes_value(true))
+        .arg(
+            Arg::with_name("tracked-shards")
+                .long("tracked-shards")
+                .takes_value(true)
+                .help("Set of shards that this node wants to track (default empty)"),
+        )
         .get_matches();
 
     let home_dir = matches.value_of("home").map(|dir| Path::new(dir)).unwrap();
     let chain_id = matches.value_of("chain-id").expect("Chain id is requried");
-    csv_to_json_configs::csv_to_json_configs(home_dir, chain_id.to_string());
+    let tracked_shards: HashSet<ShardId> = match matches.value_of("tracked-shards") {
+        Some(s) => s.split(',').map(|v| v.parse::<ShardId>().unwrap()).collect(),
```

Comment:
If someone passes an empty string, e.g. as `""` it will be `Some` and not `None`, `split(',')` will work on empty string producing an iterator over one element which is an empty string itself and `parse` will panic.

Reply:
Good point. Fixed.